app/src/main/java/org/desarrolladorslp/teckersapp/ui/parentTeckers/ParentTeckersFragment.kt

```diff
@@ -0,0 +1,66 @@
+package org.desarrolladorslp.teckersapp.ui.parentTeckers
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.desarrolladorslp.teckersapp.R
+import org.desarrolladorslp.teckersapp.model.Tecker
+import org.desarrolladorslp.teckersapp.ui.deliverables.DeliverableFragment
+import org.desarrolladorslp.teckersapp.ui.teckers.TeckerViewModel
+
+class ParentTeckersFragment : Fragment(),ParentTeckerAdapter.OnHeadlineSelectedListener{
+
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var viewManager: RecyclerView.LayoutManager
+    private lateinit var teckersViewModel: TeckerViewModel
+    private lateinit var viewAdapter: RecyclerView.Adapter<ParentTeckerAdapter.TeckerHolder>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        teckersViewModel =
+            ViewModelProviders.of(this).get(TeckerViewModel::class.java)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val root= inflater.inflate(R.layout.fragment_parent_teckers, container, false)
+        viewManager = GridLayoutManager(context,2)
+        teckersViewModel._teckers.observe(activity as AppCompatActivity, Observer{ teckers ->
+
+            viewAdapter = ParentTeckerAdapter(teckers)
+            (viewAdapter as ParentTeckerAdapter).setOnHeadlineSelectedListener(this)
+            recyclerView= root.findViewById<RecyclerView>(R.id.parentTeckersList).apply{
```

Comment:
You can use Kotlin synthetic properties here. You can do `root.parentTeckersList` instead of `root.findViewById<RecyclerView>(R.id.parentTeckersList)`